plugins/org.obeonetwork.m2doc/src/org/obeonetwork/m2doc/util/M2DocUtils.java

```diff
@@ -308,12 +308,14 @@ public static String message(ParsingErrorMessage message, Object... objects) {
      * 
```

Comment:
Please concerve old API without URIConverter. The basic user don't need to know what an URIConverter is.